### DISCUSSION TITLE
Include same directory in different locations depending on the OS

### DISCUSSION THREAD

**Discussion by cyaconi** [twpayne/chezmoi#3754] [Q&A] [answered]
I am having trouble figuring out how to include my Neovim configuration directory in different places depending on which operating system I am using for my dotfiles.
In MaOS, the configuration directory is located in `{{ homeDir }}/.config/nvim`, but in Windows, it is located in `{{ homeDir }}\AppData\Local\nvim`.
What is the idiomatic way to configure chezmoi to have the same directory in different locations depending on the operating system I am installing?
Thanks!

**Comment by bradenhilton** [comment]
https://www.chezmoi.io/user-guide/manage-machine-to-machine-differences/#handle-different-file-locations-on-different-systems-with-the-same-contents

**Comment by cyaconi** [comment]
I have read that document several times, trying to figure out how to solve my problem with that solution, but it appears to only work for files. What I need is to refer to the entire directory, including the files within it.
Is the solution to move each nvim config file to the `.chezmoitemplates` directory and then write the same directory structure in the appropriate path for both OS, with each file in those directories referencing the template?

**Reply by bradenhilton** [reply, marked as answer]
That's correct

**Reply by cyaconi** [reply]
Thank you!

**Reply by halostatue** [reply]
It's worth noting that #2273 is the eventual plan for making this cleaner.
